data/ua-update.json

```diff
@@ -48,5 +48,32 @@
   "google.at": "Mozilla/5.0 (Android 8.1.0; U; Mobile; rv:60.9) Gecko/60.9 SailfishBrowser/1.0",
   "apkpure.com": "Linux#Android 8.1.0",
   "zingmp3.vn": "Linux#Android 8.1.0",
-  "vk.com": "Linux; U; # "
+  "vk.com": "Linux; U; # ",
+  "ebay.com": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.com.au": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.at": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.be": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.ca": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.cn": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.fr": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.de": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.fr": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.com.hk": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.in": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.ie": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.it": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.co.jp": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.co.kr": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.com.my": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.nl": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.ph": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.pl": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.com.sg": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.es": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.se": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.ch": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.com.tw": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "ebay.co.th": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
+  "gittigidiyor.com": "Mozilla/5.0 (Linux; U; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0",
```

Comment:
This should not be needed I think. Loading principal of the resource should be pointing to the ebay.xyz.